net/zerotier/patches/0004-fix-compilation-for-arm_cortex-a7-neon.patch

```diff
@@ -1,7 +1,7 @@
-From 7ac88ced6cdedf88ccebfc1de12d92b269d4a60a Mon Sep 17 00:00:00 2001
+From 8a5dfa52be7f59f74616298e7d7dca20516118d2 Mon Sep 17 00:00:00 2001
 From: Moritz Warning <moritzwarning@web.de>
 Date: Mon, 30 Nov 2020 12:25:42 +0100
-Subject: [PATCH 8/8] fix compilation for arm_cortex-a7+neon
+Subject: [PATCH 4/5] fix compilation for arm_cortex-a7+neon
```

Comment:
Is this patch upstreamable? So, we don't need to carry and maintain this patch forever.

Reply:
I do not know. I would leave it as it is for now.

Reply:
I would be against that, so would it be possible to take this patch and try to submit it to the upstream repository? I don't know who the author is if you or someone else. It would help us have some references instead of looking at each OpenWrt pull request and finding the appropriate one to see where you were discussing it.

Reply:
Feel free to do yourself. I will go outside.

Reply:
Proposal for that patch: https://github.com/openwrt/packages/pull/13971#issuecomment-735690076

Reply:
@neheb would you mind to send it to upstream, please?